packages/medusa/src/api-v2/store/carts/[id]/complete/route.ts

```diff
@@ -33,5 +64,8 @@ export const POST = async (
     req.remoteQueryConfig.fields
   )
 
-  res.status(200).json({ order })
+  res.status(200).json({
```

Comment:
The fact that we return different types depending on the success is a bit strange to me. Can't we have a cart-order link, and we just return a cart here with the `order` expanded instead?

Reply:
I like the idea, we could bring this up in a future iteration. For now, its keeping in line with what we did with v1.